package/MDAnalysis/analysis/base.py

```diff
@@ -389,8 +569,151 @@ def _conclude(self):
+        self, n_parts: int, start=None, stop=None, step=None, frames=None
+    ) -> list[np.ndarray]:
+        """
+        Splits the trajectory frames, defined by `start/stop/step` or `frames`, into
+        `n_parts` even groups, preserving their indices.
+
+        Parameters
+        ----------
+        n_parts : int
+            number of parts to split the workload into
+        start : int, optional
+            start frame
+        stop : int, optional
+            stop frame
+        step : int, optional
+            number of frames to skip between each analysed frame
+        frames : array_like, optional
+            array of integers or booleans to slice trajectory; `frames` can
+            only be used *instead* of `start`, `stop`, and `step`. Setting
+            *both* `frames` and at least one of `start`, `stop`, `step` to a
+            non-default value will raise a :exc:`ValueError`.
+
+        Raises
+        ------
+        ValueError
+            if *both* `frames` and at least one of `start`, `stop`, or `frames`
+            is provided (i.e., set to another value than ``None``)
+
+        Returns
+        -------
+        computation_groups : list[np.ndarray]
+            list of (n, 2) shaped np.ndarrays with frame indices and numbers
+        """
+        if frames is None:
+            start, stop, step = self._trajectory.check_slice_indices(start, stop, step)
+            used_frames = np.arange(start, stop, step)
+        elif not all(opt is None for opt in [start, stop, step]):
+            raise ValueError("start/stop/step cannot be combined with frames")
+        else:
+            used_frames = frames
+
+        if all((isinstance(obj, bool) for obj in used_frames)):
+            arange = np.arange(len(used_frames))
+            used_frames = arange[used_frames]
+
+        # this numpy thing is similar to list(enumerate(frames))
+        enumerated_frames = np.vstack([np.arange(len(used_frames)), used_frames]).T
+
+        return np.array_split(enumerated_frames, n_parts)
+
+    def _configure_client(self, backend: str, n_workers: int, client):
+        """
+        Configure parameters necessary for running a distributed workload.
+
+        Parameters
+        ----------
+        backend : str, optional
+            scheduling type:
+              - 'dask' implies `dask` scheduler (need to install MDAnalysis[dask] for that to work)
+              - 'multiprocessing' works with standard python multiprocessing module
+              - None if you want to run things locally
+        n_workers : int, optional
+            number of workers (local or remote processes).
+
+        Raises
+        ------
+        ValueError
+            if backend is not available for current class
+
+        Returns
+        -------
+        client : ParallelExecutor
+            a ParallelExecutor instance, configured with given parameters
+        """
+        if backend is not None and backend not in self.available_backends:
```

Comment:
It missed the scenario when `backend=None` but `client=client`. Either explicitly set `backend='dask.distributed` or check for the condition here.

Reply:
done